scriptler/showAgentJavaVersion.groovy

```diff
@@ -0,0 +1,26 @@
+/*** BEGIN META {
+  "name" : "Show Agent Java Version",
+  "comment" : "It lists name and java version installed on each Agents",
+  "parameters" : [],
+  "core": "1.609",
+  "authors" : [
+    { name : "Ben Walding", "kuisathaverat" }
```

Comment:
when you run `groovy testMetaFormat.groovy` in this directory, then you see that this is not a valid format - I'll fix it and merge it